index.html

```diff
@@ -133,6 +133,7 @@
 				<li><a href="#achievements" class="smoothScroll">My Achievements</a></li>
 				<!-- <li><a href="#price" class="smoothScroll">Prices</a></li> -->
 				<li><a href="#testimonial" class="smoothScroll">Reviews</a></li>
+				>div><My Career>div>

```

Comment:
The syntax for the opening and closing div tags is incorrect. It should be "<div><My Career></div>".